backend/src/db/migrations/20230910082133_add_article_read_time_column.ts

```diff
@@ -0,0 +1,19 @@
+import { type Knex } from 'knex';
+
+const TableName = 'articles' as const;
+
+const ColumnName = 'read_time' as const;
```

Comment:
Primitive constants are already constants
```suggestion
const TABLE_NAME = 'articles';

const COLUMN_NAME = 'read_time';
```